src/components/Search/Filters/FilterOptions.tsx

```diff
@@ -0,0 +1,118 @@
+import { useRouter } from 'next/router'
+import { ReactElement } from 'react'
+import {
+  FilterByTypeOptions,
+  ResultsPerPageOptions,
+  SortByOptions,
+  SortDirectionOptions
+} from '../../../models/SortAndFilters'
+import styles from './FilterOptions.module.css'
+import classNames from 'classnames/bind'
+
+const cx = classNames.bind(styles)
+
+export type TypeKeys = keyof typeof filterTypeOptions
+
+export const filterTypeOptions = {
+  type: {
+    display: 'type',
+    options: [
+      { display: 'all', value: FilterByTypeOptions.All, default: true },
+      { display: 'algorithm', value: FilterByTypeOptions.Algorithm },
+      { display: 'dataset', value: FilterByTypeOptions.Data }
+    ]
+  },
+  sortBy: {
+    display: 'sort by',
+    options: [
+      { display: 'relevance', value: SortByOptions.Relevance, default: true },
+      { display: 'published', value: SortByOptions.Published },
+      { display: 'updated', value: SortByOptions.Updated }
+    ]
+  },
+  resultsPerPage: {
+    display: 'results per page',
+    options: [
+      { display: '10', value: ResultsPerPageOptions.Ten, default: true },
+      { display: '25', value: ResultsPerPageOptions.TwentyFive },
+      { display: '50', value: ResultsPerPageOptions.Fifty }
+    ]
+  }
```

Comment:
It seems display & value are almost always the same value. So we should probably make this an optional property.